src/bundle/Resources/config/routing.yml

```diff
@@ -1,4 +1,4 @@
-ezplatform.httpcache.invalidatetoken:
+ibexa.http_cache.invalidate_token:
     path: /_ez_http_invalidatetoken
```

Comment:
Maybe we should introduce new path here as well? But it would means that we would also need to update our vcls